buildozer/targets/osx.py

```diff
@@ -76,8 +76,6 @@ def ensure_kivyapp(self):
         else:
             self.download_kivy(kivy_app_dir, py_branch)
 
-        return
```

Comment:
I've always been bugged by that, but I think we have more in the code base here or in p4a so I didn't dare to touch it :sweat_smile: 
I'm happy to remove it still if it comes from someone else :stuck_out_tongue: